chemprop/cli/hpopt.py

```diff
@@ -43,16 +43,16 @@
     DEFAULT_SEARCH_SPACE = {
         "activation": tune.choice(categories=list(Activation.keys())),
         "aggregation": tune.choice(categories=list(AggregationRegistry.keys())),
-        "aggregation_norm": tune.quniform(lower=1, upper=200, q=1),
-        "batch_size": tune.loguniform(lower=16, upper=256, base=2),
-        "depth": tune.quniform(lower=2, upper=6, q=1),
+        "aggregation_norm": tune.qrandint(lower=1, upper=200, q=1),
```

Comment:
Why did you change this to an integer? NormAggregation expects the norm to be a float. Also looks like the `q=1` isn't needed for this and the other `qrandint`s.

Reply:
I will change it back to quniform()
Also, q = 1 is the default, so maybe we can just leave it as it is.